kornia/augmentation/container/video.py

```diff
@@ -200,11 +200,13 @@ def forward_parameters(self, batch_shape: torch.Size) -> List[ParamItem]:
                 if self.same_on_frame:
                     for k, v in mod_param.items():
                         # TODO: revise colorjitter order param in the future to align the standard.
-                        if not (k == "order" and isinstance(module, kornia.augmentation.ColorJitter)):
+                        if not (k == "order" and isinstance(module, kornia.augmentation.ColorJitter)) \
+                                and not (k == 'output_size'):
```

Comment:
I agree but I have zero experience with this video stuff. Just figured out this was needed for the tests to pass since it does not make sense to repeat this parameter. Is the other comment your suggestion how to avoid this?

Reply:
this should be handled as same as the `input_size`.

Reply:
Main differences here is `'input_size': tensor([[100, 100]]), 'output_size': tensor([200, 200])`, you may try add another dimension to see if it worked.

Reply:
I went with the same solution as input size, expanding it to the batch size. I don't like it since it would never work for different sizes in each element of the batch and one could seem to understand it could, but it does avoid the extra clause in the if-statement... Anyway, this applies also to the input size parameter so I guess it could be fixed in the future if desired, consistent for now.